src/pybind/mgr/cephadm/module.py

```diff
@@ -1416,12 +1416,19 @@ def _host_ok_to_stop(self, hostname: str) -> Tuple[int, str]:
             if dd.hostname == hostname:
                 daemon_map[dd.daemon_type].append(dd.daemon_id)
 
+        notifications = []  # type: List[str]
         for daemon_type, daemon_ids in daemon_map.items():
             r = self.cephadm_services[daemon_type_to_service(daemon_type)].ok_to_stop(daemon_ids)
             if r.retval:
                 self.log.error(f'It is NOT safe to stop host {hostname}')
                 return r.retval, r.stderr
+            if r.stdout and 'NOTICE:' in r.stdout:
```

Comment:
Would be great, if we could avoid introducing new, non-standard and undocumented serialization formats here in the mgr module.

Reply:
I think we can use directly any content in r.stdout not needed to check if it has a "NOTICE".

Reply:
I've changed it to just use all stdout. It includes the notifications and also messages about daemons that are considered okay to stop (whic I think is okay). Thoughts? @sebastian-philipp @jmolmo 
![image](https://user-images.githubusercontent.com/47704447/104335241-346e5780-54c1-11eb-8682-0bf4e5bc9a43.png)

Reply:
The only think I'm worried about is to search for `'NOTICE:'` in the output of `ok_to_stop()` instead of providing a proper return value. Just imagine someone is fixing the error message to look like

`Notice, stopping the only daemon in the Prometheus service. ...`

And suddenly this code breaks. Without any warning or make check validation. And I don't want that. That's a nightmare for future refactorization.

Reply:
@sebastian-philipp would you prefer a designated return value for providing notifications, using any stdout that comes back (as in the example output) or a different approach entirely? I can see why searching the output string is an issue and will make sure to avoid that.

Reply:
anything works for me.